questions-js1/0.md

```diff
@@ -1,6 +1,5 @@
 Q: What is a javascript REPL? Can you name two ways to open one?
 
-A: {YOUR ANSWER HERE}
+A: {It's an online javascript editor.We can open it by typing node in the terminal}
```

Comment:
a REPL is not necessarily online, since typing node in the terminal does not require an internet connection.

You can also open a REPL through the developer tools in your browser